infrastructure-playbooks/rolling_update.yml

```diff
@@ -110,10 +101,21 @@
         enabled: yes
       when: is_systemd
 
-    - name: select a running monitor
+    - name: set mon_host_count
+      set_fact: mon_host_count={{ groups.mons | length }}
+
```

Comment:
@leseb How would you like this handled? I don't think Ansible supports prompting at the task level nor having conditional prompts like this.

Also, what is the Ceph concern with updating the only monitor host? Some kind of inadvertent impact to your cluster? I can print a debug warning and/or add some sort of generic single mon host message when we initially prompt the user to confirm. But aside from that, I'm not sure what else we can do.